### DISCUSSION TITLE
selectObjectByNam

### DISCUSSION THREAD

**Discussion by fzs1933** [Stellarium/stellarium#1613] [Q&A] [answered]
Hi,

I found an oddity.
I scripted a selection command "core.selectObjectByName("Hydra")" and then want to show boundaries, lines, label and art. But Stellarium selects and shows me the Sagittarius.
Where can be the mistake?

Thanks!

**Comment by fzs1933** [comment, marked as answer]
The others (although I haven't tried all 88) work well.

**Reply by gzotti** [reply]
Pluto's moon Hydra is currently in Sagittarius. 

For constellations, prefer to use
https://stellarium.org/doc/0.21/classStelMainScriptAPI.html#ae2de9498a7272645f3b387d73c214205

**Reply by fzs1933** [reply]
Ah, I forgot that command, sorry and thanks!
